action/protocol/execution/evm/evm.go

```diff
@@ -399,9 +399,6 @@ func prepareStateDB(ctx context.Context, sm protocol.StateManager) (*StateDBAdap
 	if featureCtx.RevertLog {
 		opts = append(opts, RevertLogOption())
 	}
-	if !featureCtx.FixUnproductiveDelegates {
```

Comment:
Will removing it not cause the full sync to fail?

Reply:
no, fullsync test is done and pass